src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -118,6 +120,16 @@ export function ReceiptModal({
           <CurrencyField amount={buyAmount} token={order.outputToken} label={outputLabel} />
 
           <styledEl.FieldsWrapper>
+            {order.receiver && (
+              <styledEl.Field>
+                <FieldLabel label="Reciever" />
+                <div>
+                  <ExternalLink href={getExplorerLink(chainId, order.receiver, ExplorerDataType.ADDRESS)}>
+                    {shortenAddress(order.receiver)}
+                  </ExternalLink>
+                </div>
+              </styledEl.Field>
+            )}
```

Comment:
Can you move this field down?
I don't think it should be the first item in the list.

Maybe before or after `Order type`.

Also, can you add a tooltip?
We can start with the same as the Explorer and iterate
![image](https://github.com/cowprotocol/cowswap/assets/43217/0e1aa55d-eadb-4a9a-84be-ecc8ff796937)

Reply:
<img width="635" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/56ae5dc1-4489-44d1-9577-e036166c20be">

Makes sense. Thank you!